raiden-ts/src/path/actions.ts

```diff
@@ -20,3 +25,12 @@ export const pathFindFailed = createStandardAction('pathFindFailed').map(
 export const pfsListUpdated = createStandardAction('pfsListUpdated')<{
   pfsList: readonly Address[];
 }>();
+
+export const persistIOU = createStandardAction('persistIOU')<
+  {
+    signedIOU: Signed<IOU>;
+  },
+  ServiceId
+>();
+
+export const clearIOU = createStandardAction('clearIOU')<{}, ServiceId>();
```

Comment:
Prefer `undefined` as `payload` type, instead of empty object, as typesafe-actions has proper specialized typings for it